### ISSUE TITLE
LSP Import fails with no debugging info file

### ISSUE THREAD

**Issue by MrChristmas2000** [smeighan/xLights#268]
Here is my testing process.

Open Nutcrakcer
Select 'Sequencer' tab
Open Sequence >>> 'Winter Wonderland techno - Arches_Import_06_14_2015'
Import >> Import Effects
Select 'LSP 2.x Sequence(*.msq)'
Select the 'Winter Wonderland techno - Mega Tree NC - 06_14_2015.msq' sequence.
NOTE: This sequence was extacted from the main sequence and only has Mega Tree Channels.
when the 'Map Channels' window opens select 'Load Mapping'.
I selected my 'LSP Mapping Mega Tree MBS Test' prebuilt mapping file.
This file shows all the proper strand mappings so I click OK.
I do get the Unhandled exception error window.
If I click 'Ignore' Nutcracker comes back but if I click on any of the models strands NC goes into 'not responding' and never comes out. The other two buttons causes Nutcracker just to closes and produces no error file.
I have opened the 'Winter Wonderland techno - Mega Tree NC - 06_14_2015.msq' file with WINzip and 7 zip both and inspected each controller file. 

Each controller file will open in Notepad++ with no errors. This tells me that there is no file corruption.
Further if there was file corruption I would not have been able to open the .msq file and built the mapping file because the name of each of the 

controllers is located at the bottom of each file and the channels is distributed throughout the .msq file.

BTW can you change the lable 'LSP 2.x Sequence' to 'LSP 2.5/2.8 Sequence' because there are still some users that have never migrated up to the 2.5/8 version.

The testing files are here:
https://dl.dropboxusercontent.com/u/28041968/Nutcracker%20Testing/Winter%20Wonderland%20techno%20Mega%20Tree%20NC%20test%20files.zip

**Comment by MrChristmas2000** [comment]
Same problem with this sequence.
https://dl.dropboxusercontent.com/u/28041968/Nutcracker%20Testing/Carol%20of%20the%20Bells%20-%20Mega%20Tree%20NC%20-%2006_15_2015.zip

**Comment by MrChristmas2000** [comment]
And this sequence as well.
https://dl.dropboxusercontent.com/u/28041968/Nutcracker%20Testing/All%20I%20want%20for%20Christmas%20is%20You%20-%20Mega%20Tree%20NC%20-%2006_15_2015.zip

**Comment by dkulp** [comment]
I wasn't able to get this to crash or fail, but it does take a very long time.   Importing the megatree (24 strands * 85 nodes) takes a long time.  About 2.5 minutes on my machine.    The render after the import takes even longer, around 5.5 minutes.     This definitely shows the downside of doing node-level effects and why I strongly recommend going through the effects and changing to model or strand level effects whenever possible.  For example, the "on" effects in the first 20 seconds that are on every node should be moved to the Model level and deleted off the nodes.   The effect from 52 seconds -> 55 could be replaced with a Red on on layer 2 and a snowflake effect layered on layer 1.  The various spirals changed to actual xLights spirals.  Etc....

**Comment by MrChristmas2000** [comment]
You are running on a Mac, am I correct?

I have not gotten any of those even close to what you are experiencing. Is there something I can run on my machine to trace what is going on when the error happens?

Over time I will probably replace most of what I will be importing but I don't see me having the time to complete that and finish the hardware changes and additions to the show this year.

That may go for a lot of others as well.

I have the time now to assist with some debugging if you can work it into your busy schedule.

**Comment by MrChristmas2000** [comment]
Ok, I certainly understand the problems with memory in a 32 bit program vs a 64 bit program. So I decided to try another test. 

I cut down the Mega Tree sequence to  1/2 of the channels. Built a Nutcracker sequence to import it to and ran the test. It crashed and I have posted the new LSP sequence file and crash file here:

https://dl.dropboxusercontent.com/u/28041968/Nutcracker%20Testing/Carol%20of%20the%20Bells%20-%20Mega%20Tree%2001-12%20NC%20-%2006_15_2015.msq

https://dl.dropboxusercontent.com/u/28041968/Nutcracker%20Testing/xLights_dbgrpt-9940-20150615T160238.zip

**Comment by MrChristmas2000** [comment]
Since NC on Windows is 32bit and therefore memory limited I slept on it and today I ran a different test.
 
I was trying to convert an LSP sequence into a NC 25ms sequence using the importer so I tried doing the import into a 50ms NC sequence. Although at first try it would not convert in my whole mega tree at once, I used the two sequences where I split the mega tree strands into top and bottom sequence pieces. Those two LSP sequences then imported into NC without crashing. Yay!!! I was even able to save the NC sequence and play it.

Next I will see how far I can continue with importing my arches and other elements into the same new NC sequence. Will advise results.
